libraries/Microsoft.Bot.Builder.LanguageGeneration/Analyzer.cs

```diff
@@ -210,7 +226,7 @@ private AnalyzerResult AnalyzeExpressionDirectly(Expression exp)
                 }
                 else
                 {
-                    if (!result.TemplateReferences.Contains(templateName))
+                    if (Templates.LgOptions?.ThrowOnRecursive == true || !result.TemplateReferences.Contains(templateName))
```

Comment:
> == tru [](http://example.com/codeflow?start=61&length=7)

"== true" is not needed.

Reply:
![image](https://user-images.githubusercontent.com/11055362/131559150-bec55f70-7784-42cc-8820-515dc87a8998.png)